server/game/cards/01_SOR/units/WedgeAntillesStarOfTheRebellion.ts

```diff
@@ -0,0 +1,26 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { KeywordName, RelativePlayer, Trait } from '../../../core/Constants';
+
+export default class WedgeAntillesStarOfTheRebellion extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '4339330745',
+            internalName: 'wedge-antilles#star-of-the-rebellion',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addConstantAbility({
+            title: 'Each friendly VEHICLE unit gets +1/+1 and gains Ambush.',
```

Comment:
Our convention for trait and aspect names in player-facing text is just to do PascalCase. I have been meaning to ask the UX folks if they want to keep this convention so I'll do that soon

Reply:
I was thinking about this as I was kind of seeing where certain titles and prompts would land. I think it would be nice to be able to render these bold + italic on the frontend so either we have a library of tags and use the casing to identify them for rendering or we do something like wrapping them in brackets to make is easily parsable.

Reply:
Sgtm. I'd prefer some kind of special characters over casing, more obvious to someone reading the backend code (much harder to realize that ALLCAPS is being specially handled). Let's move the discussion over to discord and we can figure out the best option.